en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
 
-Variables keep values in memory for later access. In an expression syntax-supported input field, you can refer to these variables directly using the variable name.
+###### Examples:
 
-#### Example
+To initialize a variable of type `var` in Choreo, you can use one of the following expressions:
 
-- Converting the value of the integer variable (`total`) to a string value.
-  ```ballerina
-  total.toString()
-  ```
+```ballerina
+5
+    
+"Hello!"
+```
 
-## Supported Operators
+## Operators
 
-### String Concatenation
+### String operators
 
-You can use the `+` operator to do string concatenations. The string value can come from a string literal or a variable.
+- Concatenation
 
-#### Examples
+    You can use the `+` operator to do string concatenations. The string value can come from a string literal or a variable.
 
-- Concatenating two string literals.
- 
-  ```ballerina
-  "Hello " + "world!" 
-  ```
-  
-- Concatenating a string literal and a variable (`name`).
- 
-  ```ballerina
-  "Hi " + name
-  ```
-  
-- Concatenating two string variables (`firstName` and `lastName`).
- 
-  ```ballerina
-  firstName + lastName
-  ```
+    ###### Example:
 
-### Number Addition
-You can use the `+` operator for number addition. The number values can come from a literal or a variable.
+    To concatenate the string `Hi` and a string variable `name`, you can use the following expression:
 
-#### Examples
+    ```ballerina
+     "Hi " + name;
+    ```
 
-- Adding integer literals.
+### Arithmetic operators
 
-  ```ballerina
-  10 + 20
-  ```
-  
-- Adding decimal literals.
+Arithmetic operators perform addition, subtraction, multiplication, division, and modulo on numerical values. You can use literals or variables of type `int` or `float` as the numerical values in Ballerina expressions.
 
-  ```ballerina
-  0.56 + 1.84
-  ```
-  
-- Adding a literal and a numerical variable (`firstNumber`).
+- Addition
 
-  ```ballerina
-  firstNumber + 100
-  ```
-  
-- Adding two numerical variables (`firstNumber` and `secondNumber`).
+    You can use the `+` operator for number addition. 
 
-  ```ballerina
-  firstNumber + secondNumber
-  ```
+    ###### Example:
 
-### Number Subtraction
+    To add an integer literal (`20`) and an integer variable `number1`, you can use the following expression:
 
-You can use the `-` operator for number subtraction. The number values can come from a literal or a variable.
+    ```ballerina
+    number1 + 20;
+    ```
 
-#### Examples
+- Subtraction 
 
-- Subtracting integer literals .
+    You can use the `-` operator for number subtraction. 
 
-  ```ballerina
-  20 - 10
-  ```
-  
-- Subtracting decimal literals.
+    ###### Example:
 
-  ```ballerina
-  1.84 - 0.56  
-  ```
-  
-- Subtracting a literal from a numerical variable (`firstNumber`).
+    To subtract two integers, integer variable `yearOfBirth` from integer literal `2022`, you can use the following expression:
```

Comment:
Suggestion: "To subtract two integers (e.g., integer variable `yearOfBirth` from integer literal `2022`),  you can use the following expression:"